recipes/neuralcoref/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "neuralcoref" %}
+{% set version = "4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 23aee7418edf5c2e90b3f9f15931563b07ded8c59de386ae654b403322430ad9
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [(osx and py27) or (win and py27 and vc<14)]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - spacy >=2.1.0
```

Comment:
Is `spacy` needed in the `host` section? Usually only `pip` and `python` are enough here.

Reply:
@npavlovikj yes, you can see in the commit history i attempted to remove it and it failed on all 3 platforms as a result. see commit a4f8945